cleanup/config.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Could the /cleanup directory be in the /install directory so that the user doesn't have to copy over the config.json and variables.sh files?